queue_services/entity-filer/tests/unit/test_worker/test_correction_ben.py

```diff
@@ -0,0 +1,768 @@
+        assert changed_delivery_address.json[key] == \
+               filing['filing']['correction']['offices']['registeredOffice']['deliveryAddress'][key]
+    changed_mailing_address = Address.find_by_id(office_mailing_address_id)
+    print('changed_mailing_address: ', changed_mailing_address)
+    for key in ['streetAddress', 'postalCode', 'addressCity', 'addressRegion']:
+        assert changed_mailing_address.json[key] == \
+               filing['filing']['correction']['offices']['registeredOffice']['mailingAddress'][key]
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    'test_name, legal_type, filing_template',
+    [
+        ('ben_court_order', 'BEN', BEN_CORRECTION),
+    ]
+)
+async def test_worker_correction_court_order(app, session, mocker, test_name, legal_type, filing_template):
+    """Assert the worker process process the court order correctly."""
+    identifier = 'BC1234567'
+    business = create_entity(identifier, legal_type, 'Test Entity')
+
+    filing = copy.deepcopy(filing_template)
+
+    corrected_filing = factory_completed_filing(business, BEN_CORRECTION_APPLICATION)
+    filing['filing']['correction']['correctedFilingId'] = corrected_filing.id
+
+    file_number: Final = '#1234-5678/90'
+    order_date: Final = '2021-01-30T09:56:01+08:00'
+    effect_of_order: Final = 'hasPlan'
+
+    filing['filing']['correction']['contactPoint'] = CONTACT_POINT
+
+    filing['filing']['correction']['courtOrder'] = COURT_ORDER
+    filing['filing']['correction']['courtOrder']['effectOfOrder'] = effect_of_order
+
+    del filing['filing']['correction']['nameRequest']
+
+    payment_id = str(random.SystemRandom().getrandbits(0x58))
+    filing_id = (create_filing(payment_id, filing, business_id=business.id)).id
+
+    filing_msg = {'filing': {'id': filing_id}}
+
+    # mock out the email sender and event publishing
+    mocker.patch('entity_filer.worker.publish_email_message', return_value=None)
+    mocker.patch('entity_filer.worker.publish_event', return_value=None)
+    mocker.patch('entity_filer.filing_processors.filing_components.name_request.consume_nr', return_value=None)
+    mocker.patch('entity_filer.filing_processors.filing_components.business_profile.update_business_profile',
+                 return_value=None)
+    mocker.patch('legal_api.services.bootstrap.AccountService.update_entity', return_value=None)
+
+    # Test
+    with patch.object(NaicsService, 'find_by_code', return_value=naics_response):
+        await process_filing(filing_msg, app)
+
+    # Check outcome
+    final_filing = Filing.find_by_id(filing_id)
+    assert file_number == final_filing.court_order_file_number
+    assert datetime.fromisoformat(order_date) == final_filing.court_order_date
+    assert effect_of_order == final_filing.court_order_effect_of_order
+
+
+@pytest.mark.asyncio
+async def test_worker_proprietor_name_and_address_change(app, session, mocker):
```

Comment:
please remove this test.  BENs don't have proprietors.

Reply:
Done.